core/i18n/resources/en.js

```diff
@@ -340,6 +341,26 @@ export const enTranslation = {
     calculationSteps: 'Calculation Steps',
   },
 
+  processingStepCalculationView: {
+    attribute: 'Attribute',
+    types: {
+      quantitative: 'Quantitative',
+      categorical: 'Categorical',
+    },
+    aggregateFunction: 'Aggregate function',
+    aggregateFunctions: {
+      avg: 'Average',
+      cnt: 'Count',
+      max: 'Maximum',
+      med: 'Median',
+      min: 'Minimum',
+      sum: 'Sum',
+    },
+    confirmCancel: `**There are unsaved changes**.
```

Comment:
should we add this to common? (to be used by nodeDefView as well?)

Reply:
right